tests/cloudpickle_test.py

```diff
@@ -1307,6 +1344,7 @@ def g(x):
 class Protocol2CloudPickleTest(CloudPickleTest):
 
     protocol = 2
+    override_existing_globals = True
```

Comment:
Ok I get it now....

I find this hard to parse. I think it would be more readable and complete to loop over `[True, False]` for the two given tests.